app/controllers/vendor_api/reference_data_controller.rb

```diff
@@ -1,11 +1,11 @@
 module VendorAPI
   class ReferenceDataController < VendorAPIController
     def gcse_subjects
-      render json: { data: GCSE_SUBJECTS }
+      render json: { data: GCSE_SUBJECTS.map { |subject| subject[:name] } }
```

Comment:
Instead of mapping the `:name` keys of both `GCSE_SUBJECTS` and `A_AND_AS_LEVEL_SUBJECTS` constants, we can instead define new constants with the key mappings and define `GCSE_SUBJECTS` and `A_AND_AS_LEVEL_SUBJECTS` in their respective classes as so:
```
GCSE_SUBJECTS = GCSE_SUBJECTS_TO_CODES.map { |subject| subject[:name] }

A_AND_AS_LEVEL_SUBJECTS = A_AND_AS_LEVEL_SUBJECTS_TO_CODES..map { |subject| subject[:name] }
```
This way we wrap the old constant definitions and won't have to amend every instance of their occurrence, what do you think?

Reply:
Ok this sounds better 👍